client/scss/components/_workflow-timeline.scss

```diff
@@ -41,6 +41,9 @@
     position: relative;
     flex-shrink: 0;
     background: theme('colors.white.DEFAULT');
+    @media (forced-colors: active) {
+      background: Canvas;
```

Comment:
If possible - can this change be split to a different PR - I know the change is small but it's best if one PR can solve one problem.